### DISCUSSION TITLE
X-ray spectroscopy examples for the breakout rooms

### DISCUSSION THREAD

**Discussion by kjappelbaum** [MADICES/MADICES-2022#32] [Ideas]
I feel it could be good to rehash in a few places X-ray spectroscopy as potential example. There was a reason we initially came up with this (might even be something we can mention in the introduction talk). 

- We have experimental and simulation community 
- There seem to be some constant/well understood concepts

However, there are also examples of challenges: 

**Incentives/Benefits:**
- There is already a lot of software and data infrastucture. Additionally, there is a lot of specialized equipment. Why should one go the extra mile to convert data? 

**What needs to be standardized?:**
- Tooling: what is tooling that is needed and not dependent on the specific setup.
- What are the atomic/normalized building blocks of the workflow? 

**Fuzziness: missing data and impedance mismatch:** 
- Experimental sample is impure 
- Experimental sample might degrade under beam (beam damage) 

**Maintainability and adoption:** 
- Will the X-ray facilities maintain this? Some already have big data teams? 

I feel it could be useful for the breakout rooms to have a few case studies from the X-ray world at hand. This might help thinking about things but also might be nice examples for a potential paper. Moreover, this might also provide actionable items one can convert into code/specifications.

**Comment by ml-evs** [comment]
> **Incentives:**
> 
>     * There is already a lot of software and data infrastucture. Additionally, there is a lot of specialized equipment. Why should one go the extra mile to convert data?

I guess there's no real distinction between incentives and benefits here, but another one would be:

* If I publish my data in this interoperable way, what platforms will ingest it and what life will my data have after publication?

**Reply by helgestein** [reply]
Thinking about XRF and (in-situ) XPS data as well as maybe Raman and FTIR (disclamer: have these in my lab) there are essentially the same challenges. In XPS and XRF you even have the interesting challenge of "how" you measure it influencing what you get out i.e. different energies will yield more surface vs. buried surface data and you need to incorporate that. I could look and see if I can get some nice diverse XRF data.

**Reply by kjappelbaum** [reply]
Excellent points!

> In XPS and XRF you even have the interesting challenge of "how" you measure it influencing what you get out i.e. different energies will yield more surface vs. buried surface data and you need to incorporate that. 

This is exactly, for instance, what we hope to look at in the Fuzziness #4 breakout room. How do you map such things to computational data? 

If you could bring some data it might also make a nice case study for the hacking room #33 we're planning. Peter Murray-Rust might bring some data he extracted from the literature and one could see 
- if there is all the data needed
- if we can make some tooling (API/plotting/analysis/...) that can both (via some schema that travels with the data) that works with both your datasets
